src/models/chart-model/index.js

```diff
@@ -57,7 +58,7 @@ export default function createChartModel({
     extremumModel.command.updateExtrema(dataView, isHomeState);
     requestAnimationFrame(() => {
       // TODO: cancel requests as well to optimize???
```

Comment:
Remove the commented out code from here, and the startTime and console log.

Reply:
Yes, I will do so.

Reply:
Don't you want to keep the TODO comments as a reminder?

Reply:
doesn't matter... let's leave it as it is for now